assignments/callbacks.js

```diff
@@ -27,25 +27,37 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length)
 }
+getLength(items, function(length){
+console.log(length)})
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1])
 }
+last(items, function(lastitem){console.log(lastitem)})
```

Comment:
You consistently pass a callback function which logs to the console whatever value is passed into it. While you change the name of the parameter on each one, it is essentially the same function.

Remember, you can use functions like they're values, and save them to variables. You can then use that variable to refer to that function:

```js
function consoleLog(input) {
    console.log(input);
}

getLength(items, consoleLog);
last(items, consoleLog);
sumNums(x, y, consoleLog);
```